src/app/component/common/Card/index.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import Image from "next/image";
+import { AiOutlineHeart } from "react-icons/ai";
+import { TbCurrencyEthereum } from "react-icons/tb";
+import Button from "../../button/Button";
+
+export type CardProps = {
+  image: string;
+  title: string;
+  time: string;
+  description: {
+    by: string;
+    collateral: string;
+  };
+  bid: {
+    amount: string | number;
+    currency: string;
+  };
+  liked?: boolean;
+  onLike?: () => void;
+  buttonText?: string;
+  onRepay?: () => void;
+  isLliquidation?: boolean;
+  isCurrentBid?: boolean;
+  onButtonClick?: () => void;
+};
+
+const Card = ({
+  image,
+  time,
+  title,
+  description,
+  bid,
```

Comment:
Is better th pass a loan object here

Reply:
This card is being used for other components as well also if i pass loan object then i have to use '.' operator like loan.image, loan.time for each property of object, it is better to use primitive values than object in js